pom.xml

```diff
@@ -190,6 +188,11 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-stdlib-jdk8</artifactId>
+                <version>1.4.32</version>
```

Comment:
Add a standardized version of this as a property following the rest of the projects.